code/modules/zombie/organs.dm

```diff
@@ -0,0 +1,93 @@
+/obj/item/organ/internal/zombie_infection
+	name = "festering ooze"
+	desc = "A black web of pus and viscera."
+	parent_organ = "head"
+	slot = "zombie_infection"
+	icon_state = "blacktumor"
+	var/converts_living = FALSE
+	var/causes_damage = TRUE
+	var/datum/species/old_species = /datum/species/human
+	var/living_transformation_time = 3
+	var/revive_time_min = 450
+	var/revive_time_max = 700
+	var/timer_id
+	var/dont_delete = FALSE // to prevent set_species proc in zombify from deleting the infection organ immediately
+
+/obj/item/organ/internal/zombie_infection/New()
+	. = ..()
+	if(ishuman(owner) && !ismachine(owner) && !iszombie(owner.dna.species))
+		old_species = owner.dna.species
```

Comment:
This kind of reference isn't going to work and is going to cause circular references, please make a new PR to fix this.